app/connector/rest-swagger/pom.xml

```diff
@@ -62,6 +71,7 @@
     <dependency>
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-rest-swagger</artifactId>
+      <scope>provided</scope>
```

Comment:
Pretty sure that this is a compile time dependency, connector depends on it, it should at least be `runtime`

Reply:
PMD was complaining, but you might be right runtime should suffice